content/docs/setup/kubernetes/additional-setup/requirements/index.md

```diff
@@ -2,6 +2,8 @@
 title: Requirements for Pods and Services
```

Comment:
This document is really an up front requirement describing how things must be in order to install Istio. As such, I think this needs to be one of the first documents presented.

Reply:
That settles it then. I'll move it to the spotlight.